### PR TITLE
[FIX] Compute assets with a purchase date before the start date

### PR DESCRIPTION
**Pull request by rconjour**
[FIX] Compute assets with a purchase date before the start date

### CODE REVIEW

File: account_asset_management_xls/report/account_asset_report_xls.py [hunk 1]
```diff
@@ -648,7 +654,7 @@ def _active_report(self, _p, _xs, data, objects, wb):
         # nova start: Get all assets between dates
         cr.execute(
             "SELECT id FROM account_asset "
-            "WHERE date_start <= %s"
+            "WHERE purchase_date <= %s "
```
Comment (StefanRijnhart) [inline account_asset_management_xls/report/account_asset_report_xls.py]: There are assets in the system with date_start < purchase_date. Should we perhaps check both dates?

Comment (rconjour) [reply account_asset_management_xls/report/account_asset_report_xls.py]: Thanks, missed that part.

File: account_asset_management_xls/report/account_asset_report_xls.py [hunk 2]
```diff
@@ -648,12 +654,16 @@ def _active_report(self, _p, _xs, data, objects, wb):
         # nova start: Get all assets between dates
         cr.execute(
             "SELECT id FROM account_asset "
-            "WHERE date_start <= %s"
-            "AND (date_remove IS NULL OR date_remove >= %s) "
-            "AND id IN %s AND type = 'normal' "
+            "WHERE purchase_date <= %(period_start)s"
+            "   OR date_start <= %(period_start)s"
```
Comment (StefanRijnhart) [inline account_asset_management_xls/report/account_asset_report_xls.py]: Please combine these two clauses with parenthesis.

Comment (rconjour) [reply account_asset_management_xls/report/account_asset_report_xls.py]: Done!

**Review by StefanRijnhart — APPROVED** [review]

**Review by PieterPaulussen — APPROVED** [review]

**Pull request merged**